src/dashboard/src/components/administration/views.py

```diff
@@ -223,6 +229,160 @@ def storage(request):
     system_directory_description = 'Available storage'
     return render(request, 'administration/locations.html', locals())
 
+def usage(request):
+    usage_dirs = _usage_dirs()
+
+    return render(request, 'administration/usage.html', locals())
```

Comment:
I'd like to start moving away from using `locals()` as it's bad security, and instead of explicitly name the variables passed to the template. Can you use `{'usage_dirs': usage_dirs}`?